### DISCUSSION TITLE
Seq service could not start

### DISCUSSION THREAD

**Discussion by accessfreelancer** [datalust/seq-tickets#2200] [Help]
The seq service is not starting due to logon failure.  I dont know the password for the seq default instance account either.

How can this be resolved?  It just started to happen out of nowhere.  The local account does exist.

![image](https://github.com/datalust/seq-tickets/assets/51207026/ee444eef-d437-42da-b583-08edb236c7c8)

![image](https://github.com/datalust/seq-tickets/assets/51207026/dcd54a34-8e7c-4864-9a88-035ab24b4b59)

**Comment by nblumhardt** [comment]
Hi! Not sure what will have happened, but you should be able to resolve this by running Seq Service Administration from the Start menu, ensuring all of the fields have the values you expect. The final configuration stage of the wizard will reset this account's password and restart the service.

HTH,
Nick
